packages/vscode-extension/src/webview/views/DeviceLocationView.css

```diff
@@ -51,30 +51,3 @@ input.coordinate:disabled {
 .supported-formats li {
   margin-bottom: 5px;
 }
-
```

Comment:
moved to https://github.com/software-mansion/react-native-ide/pull/422/files#diff-e73cae9b9a44401cdd88d9db6605d882d057713ccd90af8482aa2b5dcef581b7